pkgs/tools/system/thinkfan/default.nix

```diff
@@ -1,18 +1,27 @@
-{ stdenv, fetchurl, cmake
+{ stdenv, fetchFromGitHub, cmakeCurses, libyamlcpp, pkgconfig
```

Comment:
Why do you need curses support in cmake?

Reply:
Thanks for your feedback! I pushed the requested changes.